src/main/java/de/rabea/game/Game.java

```diff
@@ -46,7 +27,7 @@ public void play(int boardDimension) {
         }
         finishGame(currentPlayer.mark(), board);
         if (userInterface.playAgain()) {
-            setUpNewGame();
+            gameSetup.setUpNewGame();
```

Comment:
Why would the game need to start 'a new game'?
Is the `Game` class responsible for running _a single game_ or _all of them_?